plugin/evm/block.go

```diff
@@ -130,21 +127,6 @@ func (b *Block) handlePrecompileAccept(rules *params.Rules, sharedMemoryWriter *
 		}
 	}
 
-	// If Warp is enabled, add the block hash as an unsigned message to the warp backend.
-	if rules.IsPrecompileEnabled(warp.ContractAddress) {
-		blockHashPayload, err := payload.NewHash(ids.ID(b.ethBlock.Hash()))
-		if err != nil {
-			return fmt.Errorf("failed to create block hash payload: %w", err)
-		}
-		unsignedMessage, err := avalancheWarp.NewUnsignedMessage(b.vm.ctx.NetworkID, b.vm.ctx.ChainID, blockHashPayload.Bytes())
-		if err != nil {
-			return fmt.Errorf("failed to create unsigned message for block hash payload: %w", err)
-		}
-		if err := b.vm.warpBackend.AddMessage(unsignedMessage); err != nil {
-			return fmt.Errorf("failed to add block hash payload unsigned message: %w", err)
-		}
-	}
-
```

Comment:
Nope this PR changes how it serves those signatures to serve them by looking them up based off of the block hash rather than putting them in the warp backend.

Reply:
but that eventually means we won't be putting them into the backend for "all" accepted blocks. (unless specifically requested?)

Reply:
No, we never put a block hash payload in the backend. We will sign any accepted block hash: https://github.com/ava-labs/subnet-evm/blob/7db675f7bb9af770ac45fc72f3859c1f69df471c/warp/backend.go#L132